pilot/pkg/networking/core/v1alpha3/listener_address.go

```diff
@@ -51,11 +52,15 @@ func getActualWildcardAndLocalHost(node *model.Proxy) (string, string) {
 	return WildcardIPv6Address, LocalhostIPv6Address
 }
 
-func getPassthroughBindIP(node *model.Proxy) string {
-	if node.SupportsIPv4() {
-		return InboundPassthroughBindIpv4
+func getPassthroughBindIPsForDualStack(ipMode model.IPMode) []string {
```

Comment:
```suggestion
func getPassthroughBindIPs(ipMode model.IPMode) []string {
```

no need to have `DualStack` appeneded

Reply:
okay, remove it

Reply:
Hi @jacob-delgado, done, removed prefix `ForDualStack` for  `getPassthroughBindIPs` and `getWildcardsAndLocalHost`